examples/Multi_Target/pareto.py

```diff
@@ -0,0 +1,170 @@
+## Single-Target vs. Pareto Optimization
+
+# In this example, we illustrate the difference between single-target and Pareto
+# optimization under laboratory conditions provided by a pair of synthetic targets.
+# In particular:
+# * We set up two root mean square target functions with different center points,
+# * visualize the corresponding Pareto frontier,
+# * and compare the recommendations obtained by optimizing the targets individually
+#   and jointly.
+
+
+### Imports
+
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+from polars import set_random_seed
+
+from baybe.acquisition import qLogNEHVI
+from baybe.campaign import Campaign
+from baybe.objectives import ParetoObjective
+from baybe.parameters import NumericalContinuousParameter
+from baybe.recommenders import BotorchRecommender
+from baybe.searchspace import SearchSpace
+from baybe.targets import NumericalTarget
+from baybe.utils.dataframe import arrays_to_dataframes
+
+### Settings
+
+# Let's first define some general settings for our example:
+
+BATCH_SIZE = 10
+N_TRAINING_DATA = 100
+N_GRID_POINTS = 100
+CENTER_Y0 = (-0.5, -0.5)
+CENTER_Y1 = (0.5, 0.5)
+
+# Also, we fix the random seed for reproducibility:
+
+set_random_seed(1337)
+
+
+### Defining the Optimization Problem
+
+# We start by defining the parameters and targets describing the inputs and outputs
+# of our synthetic functions:
+
+x0 = NumericalContinuousParameter("x0", (-1, 1))
+x1 = NumericalContinuousParameter("x1", (-1, 1))
+y0 = NumericalTarget("y0", "MIN")
+y1 = NumericalTarget("y1", "MIN")
+searchspace = SearchSpace.from_product([x0, x1])
+
+# With these definitions at hand, we can construct a multi-variate callable representing
+# the two quadratic target functions:
+
+
+@arrays_to_dataframes([x0.name, x1.name], [y0.name, y1.name])
+def lookup(arr: np.ndarray) -> np.ndarray:
+    """Compute root mean square values for different center points."""
+    y0 = np.sqrt(np.sum((arr - CENTER_Y0) ** 2, axis=1))
```

Comment:
As mentioned earlier, we could also change this from RMSE to MSE, in order to get a curved front in target space. Let me know what you prefer

Reply:
Have dropped the square root to show the curvature